Content.Client/Crayon/CrayonComponent.cs

```diff
@@ -10,5 +10,8 @@ public sealed partial class CrayonComponent : SharedCrayonComponent
         [ViewVariables(VVAccess.ReadWrite)] public bool UIUpdateNeeded;
         [ViewVariables] public int Charges { get; set; }
         [ViewVariables] public int Capacity { get; set; }
+        [ViewVariables] public string State { get; set; }
```

Comment:
this property appears to be a duplicate of `SharedCrayonComponent.SelectedState`